content/etc/icingaweb2/config.ini

```diff
@@ -0,0 +1,10 @@
+[global]
+show_stacktraces = "1"
+config_backend = "db"
+config_resource = "icingaweb_db"
+module_path = "/usr/share/icingaweb2/modules:/etc/icingaweb2/modules"
```

Comment:
No, the dockerfile currently uses the official module. I thought about using findmypast's module, but it has got some technical disadvantages.